src/irr/asset/CImageLoaderTGA.cpp

```diff
@@ -115,22 +114,20 @@ bool CImageLoaderTGA::isALoadableFileFormat(io::IReadFile* _file) const
 
 // convertColorFlip() does color conversion as well as taking care of properly flipping the given image.
 template <typename T, E_FORMAT srcFormat, E_FORMAT destFormat>
```

Comment:
why is there a template on T, why are you incrementing the `in` and `out` by the same stride that is in pixels and not in bytes?

Reply:
because somebody created it, but if it comes to stride, I need to fix it

Reply:
just use uint8_t* always.